### DISCUSSION TITLE
map scrolling across date line

### DISCUSSION THREAD

**Discussion by edst42** [mapsforge/mapsforge#1208] [Ideas]
Hi,
I am writing an app using mapsforge to display a world map. (The world map I generate as an .osm map and from that with the help of osmosis into the .map file used by mapsforge.)
The map is shown as the typical Mercator projection with 0,0 as map center with the left/right edges at the (I guess) "date line" (-180/180 degrees longitude).
I would like to be able to also see the pacific ocean at the map center. So ideally I would like to be able to scroll across that 180/-180 line. So basically not seeing a map boundary at the left and right edges.
I could not find anything about this in the Mapsforge documentation so I am guessing this is not possible.
Is that right?

Kind regards,
Ed

**Comment by devemux86** [comment]
This is not available in Mapsforge.

It is available in VTM OpenGL map library: https://github.com/mapsforge/vtm

**Reply by edst42** [reply]
Thank you, I will check VTM.
EDIT: I prefer to stick to mapsforge for the moment, given that I already invested effort in it. 
I do have another question about Mapsforge rendering about node display. I will post a separate question for it.
